content/en/docs/contributing/google-season-of-docs/improve-navigation-and-structure/index.md

```diff
@@ -0,0 +1,22 @@
+# Proposal: Improve the navigation structure of the cert-manager website
+
+Right now the content is not designed with our target audiences in mind.
```

Comment:
Note to self: so this aims to introduce the 'use case based documentation' that has been talked about previously.